sig-contributor-experience/role-handbooks/apac-coordinator.md

```diff
@@ -0,0 +1,25 @@
+# Contributor Experience APAC Coordinator
+
+## Overview
+
+The Kubernetes contributor base is still heavily skewed towards the US, and we are trying to diversify geographically. APAC Coordinator in Contributor Experience SIG helps to encourage and inspire more people in APAC regions to start contributing and ensure that there is adequate representation from there.
+
+Some common tasks include:
+
+- help with mirroring programs and strategies for contributors there
+- speak at meetups and local events about the work that happens in contributor experience
+- write blog posts about the same in multiple languages, coordinate translation when necessary
+- help with regional boards on discuss.kubernetes.io
+- help with regional channels on Slack
+- run the APAC Contributor Experience update meeting
+
+APAC Coordinators currently meet at [6:00 UTC](http://www.thetimezoneconverter.com/?t=06:00&tz=UTC) on Thursday every two weeks starting from June 13th, the [meeting notes doc](https://docs.google.com/document/d/1qf-02B7EOrItQgwXFxgqZ5qjW0mtfu5qkYIF1Hl4ZLI/) is the same one as the SIG's.
```

Comment:
We'll try to make this meeting as regular as possible, and it may eventually be replaced by the APAC version of the SIG meeting. Adding it to sigs.yaml also means we might want to invite the sig mailing list instead of just people who involved so far.